### ISSUE TITLE
Webpack issue in vuejs

### ISSUE THREAD

**Issue by satkunas** [pytesNET/tail.DateTime#35]
Excellent project. We're attempting to get jquery out of our stack and are hoping to use this as a dropin replacement for `pc-bootstrap4-datetimepicker`. However we've run into a context issue when `tail.datetime` is used within VueJS.

Env:
- bootstrap: 4.3.1
- bootstrap-vue: 2.0.0-rc.19
- vue: 2.6.10
- webpack 4.28.4

Usage:
```js
// import
const datetime = require('tail.datetime')
// invoke
datetime('.datetime', config)
```

Error:
```
TypeError: Cannot read property 'querySelectorAll' of undefined
```

- The error occurs at https://github.com/pytesNET/tail.DateTime/blob/master/js/tail.datetime-full.js#L78
- This is because `d` is undefined. 

Issue:
When invoked from within a VueJS component, `root` is {}, not `window` as expected.

Fix:
 Change the line https://github.com/pytesNET/tail.DateTime/blob/master/js/tail.datetime-full.js#L37

from
```js
}(this, function(root){
```
to 
```js
}(window, function(root){
```

**Comment by satkunas** [comment]
@SamBrishes I was hoping to hear back from you on this in the past week. Any chance for a fix?

**Comment by SamBrishes** [comment]
Hellow,

I'm really sorry for the long wait, I had some "private difficulties".

Thanks for your Issue / Bug Report and the solution (I'm not into VueJS / Webpack). Of course, I'll change `this` into `window` in the next update (0.4.13), which should be released very soon (It contains also some other important fixes / changes).

Thanks.

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

sorry for the long wait, the issue has been fixed in Version 0.4.13.

Thanks for your contribution!

Sincerely,
Sam.
